src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -2631,10 +2631,10 @@ public void removeCommentHelper(Element element) {
 
     public List<? extends DocTree> getBlockTags(Element element, Taglet taglet) {
         return getBlockTags(element, t -> {
-            if (taglet instanceof BaseTaglet) {
-                return ((BaseTaglet) taglet).accepts(t);
-            } else if (t instanceof UnknownBlockTagTree) {
-                return ((UnknownBlockTagTree) t).getTagName().equals(taglet.getName());
+            if (taglet instanceof BaseTaglet bt) {
+                return bt.accepts(t);
+            } else if (t instanceof UnknownBlockTagTree ukbt) {
+                return ukbt.getTagName().equals(taglet.getName());
```

Comment:
Use `tree` 

`ukbt` is not even an acronym, unless you spell it Un Known Blocktag Tree!